presto-parquet/src/main/java/io/prestosql/parquet/reader/ParquetReader.java

```diff
@@ -270,4 +297,9 @@ public AggregatedMemoryContext getSystemMemoryContext()
     {
         return systemMemoryContext;
     }
+
+    public long getMaxCombinedBytesPerRow()
```

Comment:
I could be missing it, but I don't see any uses of this method in the diff.

Reply:
Nice catch. I was thinking about updating the `FileFormatDataSourceStats` but the change is non-trivial (Parquet reader has the stats object in ParquetDataSource, therefore, it requires changes to the interface). I'll remove it for now. We can address it later if we need this stats.